tests/commands/client_test.py

```diff
@@ -120,12 +120,12 @@ def test_http_get(self):
     def test_action_runs(self):
         self.client.action_runs('/api/jobs/name/0/act', num_lines=40)
         self.client.request.assert_called_with(
-            '/api/jobs/name/0/act?include_stdout=1&num_lines=40&include_stderr=1')
+            '/api/jobs/name/0/act?num_lines=40&include_stdout=1&include_stderr=1')
```

Comment:
Instead of pinning hash seed and depending on "random"ness, I'd rather these either:
- produce deterministic results
- compare hash-agnostically